app/kontainerdriver_data.go

```diff
@@ -45,6 +45,17 @@ func addKontainerDrivers(management *config.ManagementContext) error {
 		return err
 	}
 
+	if err := creator.addCustomDriver(
+		"baiducloudcontainerengine",
+		"https://github.com/cnrancher/kontainer-engine-driver-baidu/releases/download/v0.2.0/kontainer-engine-driver-baidu-linux",
+		"4613e3be3ae5487b0e21dfa761b95de2144f80f98bf76847411e5fcada343d5e",
+		"https://cluster-driver.oss-cn-shenzhen.aliyuncs.com/baidu/ui/component.js",
+		false,
+		"*.aliyuncs.com", "*.baidubce.com",
+	); err != nil {
```

Comment:
Why there is `*.aliyuncs.com` stuff here?

Reply:
We put UI package on aliyun OSS, so we need to add `*.aliyuncs.com` to whitelist domain.